src/react/App.tsx

```diff
@@ -10,6 +11,7 @@ export default function App() {
         <title>Bun, Elysia & React</title>
         <meta name="description" content="Bun, Elysia & React" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="./public/assets/styles.css" />
```

Comment:
The tricky part is that our websites static files are prefixed with `./public/`, so make sure to prepend to the path when using!